Cargo.toml

```
@@ -167,8 +167,8 @@ ya-sb-router = { path = "service-bus/router" }
 ya-sb-util = { path = "service-bus/util" }
 
 ## CLIENT
-ya-client = { git = "https://github.com/golemfactory/ya-client.git", rev = "fe426adc89682be018d65504c0bfb8ab9580fcae"}
-ya-client-model = { git = "https://github.com/golemfactory/ya-client.git", rev = "fe426adc89682be018d65504c0bfb8ab9580fcae"}
+ya-client = { git = "https://github.com/golemfactory/ya-client.git", rev = "103beac31f563c72d8c9c52f7f2f4f84176eab1f"}
```

Its cleaner to use the merge commit instead of the last PR, you. might accedentilly revert other commits on master this way.
```suggestion
ya-client = { git = "https://github.com/golemfactory/ya-client.git", rev = "80221a38c4cf8f4700da36598316fc8048e5c535"}
```

---

Ok, we will upgrade it before merge to master.
But we wanted to have build as fast as possible to update devnet-alpha.3

---

agree with updating the devnet asap :D

I can also update the SHA in my zksync PR, one final test ( including this fix ) and it is good to go.